adoc/admin-security-rbac.adoc

```diff
@@ -139,7 +139,7 @@ https://github.com/dexidp/dex/tree/v2.16.0/Documentation/connectors.
 . Apply changes:
 +
 ----
-kubectl apply -f ~/clusters/<CLUSTER_NAME>/addons/dex/dex.yml
+kubectl apply -k ~/clusters/<CLUSTER_NAME>/addons/dex/
```

Comment:
There is no need to do this, `skuba` will [do this automatically](https://github.com/SUSE/skuba/blob/878d9b8b3a0812897e8ce60d099ad12d834606c0/internal/pkg/skuba/addons/addons.go#L336).

[1]

Reply:
This is for adapting changes later